apps/client/src/intros/DakraamIntro.js

```diff
@@ -1,42 +1,19 @@
-import { Heading, Paragraph } from "@datapunt/asc-ui";
 import React from "react";
 
-import { List, ListItem } from "../atoms";
-import ContactSentence from "../components/ContactSentence";
-import { eventNames } from "../config/matomo";
-import { IntroHeading, IntroSituationDescription } from "./shared";
+import { Intro } from "./shared";
 
 export default () => (
-  <>
-    <IntroHeading />
-
-    <List variant="bullet">
-      <ListItem>het wijzigen van een monument.</ListItem>
-      <ListItem>het plaatsen van het dakraam zelf.</ListItem>
-    </List>
-
-    <IntroSituationDescription />
-
-    <Paragraph>
-      Deze vergunningcheck gaat over dakramen, daklichten en lichtstraten. U
-      kunt hem gebruiken als u een nieuwe plaatst of als u een bestaande
-      vernieuwt.
-    </Paragraph>
-
-    <Heading forwardedAs="h4">Uitzonderingen:</Heading>
-    <Paragraph gutterBottom={8}>
-      In de volgende situaties is het niet mogelijk de vergunningcheck te
-      gebruiken:
-    </Paragraph>
-    <List variant="bullet" style={{ marginBottom: 12 }}>
-      <ListItem>Het aantal woningen verandert.</ListItem>
-      <ListItem>
-        Het deel van het gebouw waarin het dakraam komt, is zonder vergunning
-        gebouwd.
-      </ListItem>
-    </List>
-    <Paragraph>
-      <ContactSentence eventName={eventNames.INTRO_EXTRAORDINARY} />
-    </Paragraph>
-  </>
+  <Intro
+    usableForBullets={[
+      "het wijzigen van een monument.",
+      "het plaatsen van het dakraam zelf.",
+    ]}
+    usableForText={`Deze vergunningcheck gaat over dakramen, daklichten en lichtstraten. U
+    kunt hem gebruiken als u een nieuwe plaatst of als u een bestaande
+    vernieuwt.`}
+    exceptions={[
```

Comment:
Same for `exceptions`

Reply:
It's on purpose. The Intro component is introduced by sttr-refactor. Now this intro page can use that same component.